tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_sms_report.py

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+from marionette_driver import expected, By, Wait
+from gaiatest import GaiaTestCase
+from gaiatest.apps.base import Base
+from gaiatest.apps.messages.app import Messages
+
+
+class TestSmsReport(GaiaTestCase):
+
+    def test_sms_report(self):
+        
+        _text_type = (By.CSS_SELECTOR, 'p[data-l10n-id="message-type-sms"]')
+        _sender = (By.CSS_SELECTOR, 'a[data-name]')
+        _receiver = (By.CSS_SELECTOR, 'a.suggestion.js-contact-info')
+        
+        time_sent = time.time()
+        _text_message_content = "Automated Test %s" % str(time_sent)
+        _test_phone_number = self.environment.phone_numbers[0]
+
+        error_range_sent_in_seconds = 120
+        error_range_received_in_seconds = 300
+        
+        # Sending the message
+        self.messages = Messages(self.marionette)
+        self.messages.launch()
+        new_message = self.messages.create_new_message(recipients=_test_phone_number,
+                                                       message=_text_message_content)
+
+        self.message_thread = new_message.tap_send()
+        self.message_thread.wait_for_received_messages()
+
+        last_received_message = self.message_thread.received_messages[-1]
+        last_message = self.message_thread.all_messages[-1]
+        
+        report = last_received_message.open()
+
+        # Checking that the data in the received report is correct
+        self.assertTrue(report.is_message_an_sms)
+        self.assertEqual(_test_phone_number, report.sender)
+
+        time_sent_err = (time_sent + error_range_sent_in_seconds) 
+        self.assertLessEqual(report.sent_date, time_sent_err)
```

Comment:
Nit: comparing date vs time sounds odd. I'd rename the variables to make sure we test the same thing (a `date_time` or a `timestamp`).